MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -717,4 +813,28 @@ p{
   .social-media{
     border: 0px;
   }
+
+  .title-icon{
+    top: 50px;
+  }
+
+  .promotion-message{
```

Comment:
1. missing space before `{`
2. Please reorder the properties in alphabetical order